resources/ottawa.csv

```diff
@@ -1,7 +1,7 @@
+capabilities,"vm:standard,arch:32bit,os:win,arch:x86,test:java9","vm:standard,arch:64bit,os:win,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:win,arch:x86,test:java9",,"vm:standard,arch:32bit,os:linux,os:unix,arch:x86,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:standard,arch:32bit,os:linux,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:ppc,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:ppcle,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:ppc,arch:ppcle,test:java9",,"vm:standard,arch:31bit,os:linux,os:unix,arch:390,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:znext,test:java9",,,"vm:standard,arch:32bit,os:aix,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:aix,os:unix,arch:ppc,test:java9","vm:compressed_refs,arch:64bit,os:aix,os:unix,arch:ppc,test:java9",,"vm:standard,arch:31bit,os:zos,arch:390,test:java9","vm:standard,arch:31bit,os:zos,arch:znext,test:java9","vm:standard,arch:64bit,os:zos,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:zos,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:zos,arch:znext,test:java9","vm:mixed_refs,arch:64bit,os:zos,arch:390,test:java9","vm:mixed_refs,arch:64bit,os:zos,arch:znext,test:java9","arch:32bit,os:linux,arch:arm","vm:standard,arch:64bit,os:osx,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:osx,os:unix,arch:x86,test:java9",,"vm:standard,arch:64bit,os:osx,os:unix,arch:aarch64,test:java9",,,,"vm:standard,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9",,"vm:standard,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9"

```

Comment:
Sorry, it is hard for me to see. What was changed here?

Reply:
Renamed `arch:zartemis` to `arch:znext`.